Source/JavaScriptCore/runtime/CachedTypes.cpp

```diff
@@ -1817,8 +1817,8 @@ class CachedFunctionExecutable : public CachedObject<UnlinkedFunctionExecutable>
 
     unsigned m_firstLineOffset : 31;
     unsigned m_lineCount : 31;
+    unsigned m_unlinkedFunctionStart : 31;
     unsigned m_isBuiltinFunction : 1;
-    unsigned m_unlinkedFunctionNameStart : 31;
```

Comment:
Let's keep it here. Otherwise, it enlarges the size of this struct (`31` and `1` is paired).

Reply:
Ah yes, I guess this change would forbid the next field (`m_isBuiltinFunction`) to be packed (as the previous takes 31 bits). Thanks!